src/react/ui-elements/Veeam/VeeamConfiguration.tsx

```diff
@@ -142,7 +142,10 @@ const Configuration = () => {
       bucketName,
       application,
       capacityBytes: getCapacityBytes(capacity, capacityUnit),
-      enableImmutableBackup,
+      enableImmutableBackup:
+        application === VEEAM_BACKUP_REPLICATION_XML_VALUE
+          ? enableImmutableBackup
+          : undefined,
```

Comment:
I think `enableImmutableBackup` is a boolean ?
```suggestion
          : false,
```